compiler/codegen.js

```diff
@@ -4137,6 +4197,42 @@ const generateSwitch = (scope, decl) => {
 
   depth.push('switch');
 
+  if (
```

Comment:
I thought we agreed to remove all this for now/this PR?

Reply:
Did you mean the stuff for `generateSwitch` too? That's the meat of this PR in all honesty

Reply:
I may have missed where this was oops, nvm!